Code/wwphys/Path.cpp

```diff
@@ -1156,7 +1156,12 @@ PathClass::Initialize_Human_Spline(DynamicVectorClass<PATH_NODE> &node_list)
 		//	Add this point as a key along the spline
 		//
 		current_dist		+= (point - last_point).Length ();
-		float curr_time	= current_dist / m_TotalDist;
+		float curr_time;
+		if (m_TotalDist != 0.f) {
```

Comment:
Nitpick, but inconsistent foating point literal format, 0.0f or 0.f, pick one? I assume the 0.f is a typo and is supposed to be 0.0f?

Reply:
Yes, `0.0f` is also the format used by the code throughout.